pgctl/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+__version__ = '0.2.1'
```

Comment:
Please DRY this versus setup.py.

Here's the official PYPA advice:

https://packaging.python.org/en/latest/single_source_version/#single-sourcing-the-version